src/main/java/com/openlattice/datastore/pods/DatastoreMvcPod.java

```diff
@@ -72,7 +71,8 @@
                 EdmController.class, OrganizationsController.class,
                 DataStoreExceptionHandler.class, EntitySetsController.class,
                 RequestsController.class, AppController.class, GraphController.class,
-                PersistentSearchController.class, AdminController.class, AssemblyAnalyzationController.class,
+                PersistentSearchController.class, AdminController.class,
+//                AssemblyAnalyzationController.class,
```

Comment:
can we delete this safely?

Reply:
Yeah as far as I know its currently unused. Just commented it for PR to flag for discussion